articles/Setup-Linux-Python.md

```diff
@@ -48,13 +53,34 @@ On Ubuntu 16.04 install it like this:
 sudo apt-get install openmpi-bin
 ```
 
-Make sure that it's libraries can be found, e.g., by setting up `LD_LIBRARY_PATH`.
+Make sure that its libraries can be found, e.g., by setting up `LD_LIBRARY_PATH`.
 
 ## Installing CNTK for Python on Linux (Ubuntu 14.04 or Ubuntu 16.04)
```

Comment:
The master branch only supports ubuntu 16.04. ubuntu 14.04 is for older versions